apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/WebviewErrorDrawer/index.tsx

```diff
@@ -7,6 +7,7 @@ import { useGetSwapTrackingProperties } from "../../utils/index";
 import { BoxedIcon, Text } from "@ledgerhq/react-ui";
 import { SwapLiveError } from "@ledgerhq/live-common/exchange/swap/types";
 import ErrorIcon from "~/renderer/components/ErrorIcon";
+import { track } from "~/renderer/analytics/__mocks__/segment";
```

Comment:
```suggestion
import { track } from "~/renderer/analytics/segment";
```

I think import is not correct here